dicom-archive/DICOM/DICOM.pm

```diff
@@ -170,13 +266,33 @@ sub setIndex {
   return 1;
 }
 
-# Return sort index.
+
+=pod
+
+=head3 getIndex()
+
+Returns the sort index.
+
+RETURNS: sort index
+
+=cut
 
 sub getIndex {
   return $sortIndex;
 }
 
-# Return value of the element at (group, element).
+
+=pod
+
+=head3 value($gp, $el)
+
+Return value of the element at (group, element).
```

Comment:
returns instead of return (although I do not like that we describe the function bu what it returns and then a line later we have a return:))